### ISSUE TITLE
Where is the pretraining weight of the biHome loss model?

### ISSUE THREAD

**Issue by Jerry-ojk** [NeurAI-Lab/biHomE#2]
Where is the pretraining weight biHome loss model? Can you offer a pretraining weight of the biHome loss model? I don't know how to train the loss network alone.

**Comment by dkoguciuk** [comment]
Hi @Jerry-ojk ,

unfortunately, I am not able to share trained models, however, it is really easy to reproduce it. I suggest you use Zeng backbone + biHomE loss trained on the PDS-COCO dataset since this is the most robust configuration:

python3 train.py --config_file config/pds-coco/zeng-bihome-lr-1e-3.yaml

In general, you don't need to train the loss network alone -- as a first 







Best,
Daniel

**Comment by dkoguciuk** [comment]
Hi @Jerry-ojk ,

unfortunately, I am not able to share trained models, however, it is really easy to reproduce them. I suggest you use Zeng backbone + biHomE loss trained on the PDS-COCO dataset since this is the most robust configuration:

python3 train.py --config_file config/pds-coco/zeng-bihome-lr-1e-3.yaml

In general, you don't need to train the Loss Network alone -- at first, I would suggest using an off-the-shelf pre-trained model like ResNet34. More details on what is important in Loss Network you can find in the paper: https://openaccess.thecvf.com/content/CVPR2021W/IMW/html/Koguciuk_Perceptual_Loss_for_Robust_Unsupervised_Homography_Estimation_CVPRW_2021_paper.html

Please let me know if you have any more questions,
Daniel

**Comment by dkoguciuk** [comment]
Closing, because of lack of activity. Please feel free to reopen if you still need help with reproduction.
